PLCSystemLogic.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collection.Generic;
+
+namespace mujinplccs
+{
+    public class MujinSystem{
+        List<Location> locations;
+        public void AddLocation(PLCLocation location){  // user call system.AddLocation(location1) to add location, when system start and location queue is not empty, will automatics start Cycle after condition satisifed
+            this.location.Insert(location);
```

Comment:
this.locations

Reply:
Would recommend to use `VisualStudio Code` instead of `vim`, or at least `vim` with some IDE functionalities

Reply:
please also correct english in all comments...